### DISCUSSION TITLE
Event seen in Kibana, but not SO's Alerts or Hunt

### DISCUSSION THREAD

**Discussion by jmptbl** [Security-Onion-Solutions/securityonion#1824] [Unsupported Versions] [answered]
Hello

Is there a reason an event would display in the Kibana interface, but be invisible to SO's Alerts and Hunt section? In Hunt it even seems impossible to search for these invisible events.

**Comment by jertel** [comment]
Hi, in Hunt, please try switching to Absolute Time mode and change the ending time to 1 hour in the future and report back if the data shows up.

**Reply by jmptbl** [reply]
@jertel Yes, that works. Thanks! I'm guessing a timezone issue? SO's interface seems to detect my timezone correctly, but I'm guessing it isn't accounting for DST.

**Comment by jertel** [comment, marked as answer]
Thanks for getting that info back to me. We will investigate further.

**Reply by jertel** [reply]
Tracked down this issue to be caused by a Go daylight savings time/timezone issue. I'm aiming to have it updated in the next release.

Golang bug details are here: https://github.com/golang/go/issues/42138
